src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { useHistory, useParams } from 'react-router';
+import { AccountIssuedTokenTable } from '../AccountIssuedTokenTable';
+import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable';
+
+// TODO: Add state types or convert to react query
+interface Props {
+  account: any;
+}
+
+const AccountAssetTabDisconnected = ({ account }: Props) => {
+  const assetTypes = ['issued', 'nft'];
+  const { id: accountId, assetType = assetTypes[0] } = useParams<{
+    id: string;
+    assetType: string;
+  }>();
+  const supportsNFT = ['nft_sandbox', 'devnet'].includes(
+    process.env.REACT_APP_ENVIRONMENT as string
+  );
```

Comment:
a superior way to do this would be to check what amendments are enabled by the connected server using `ledger_entry`

Reply:
Agreed.  I think that should happen in another phase and we will at that point also add a page to show all amendments enabled on a network.  Because of the async nature of such a thing we would need to look at rewriting some stuff.